src/components/Auth/Login.js

```diff
@@ -1,6 +1,53 @@
+import axios from "axios";
 import React from "react";
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import { useAuthContex } from "../../contex/auth-contex";
+
 import "./auth.css";
 const Login = () => {
+  const navigateHome = useNavigate();
+  const { authDispatch } = useAuthContex();
+  const [userDetail, setUserDetail] = useState({
+    email: "",
+    password: "",
+  });
+  const testUser = {
+    email: "adarshbalak@gmail.com",
+    password: "adarshBalaki123",
+  };
+  const setUserDetailHandler = (e) => {
+    let name = e.target.name;
+    let value = e.target.value;
+    setUserDetail({ ...userDetail, [name]: value });
+  };
+  const testCredentials = (e) => {
+    e.preventDefault();
+    setUserDetail(testUser);
+  };
+  const submitUserDetail = async (e) => {
+    e.preventDefault();
+    try {
+      const response = await axios.post("/api/auth/login", {
+        email: userDetail.email,
+        password: userDetail.password,
+      });
```

Comment:
As you are already having your `userDetail` as an object, so you can simply pass that to your function, it will reduce your code:- `const response = await axios.post("/api/auth/login",userDetail)`.